sway-lib-std/src/u128.sw

```diff
@@ -46,9 +46,20 @@ impl core::ops::Ord for U128 {
     }
 }
 
-// TODO this doesn't work?
-// impl core::ops::OrdEq for U128 {
+// trait OrdEq {
+// } {
+//     fn ge(self, other: Self) -> bool {
+//         self.gt(other) || self.eq(other)
+//     }
+
+//     fn le(self, other: Self) -> bool {
+//         self.lt(other) || self.eq(other)
+//     }
 // }
+
+impl core::ops::OrdEq for U128 {
```

Comment:
I believe the reason why CI continues to fail is due to https://github.com/FuelLabs/sway/issues/1667. If so, then we cannot implement `OrdEq` for `U128` just yet.